src/features/unified-share-modal/InviteePermissionsMenu.js

```diff
@@ -66,8 +67,15 @@ class InviteePermissionsMenu extends Component<Props> {
     }
 
     render() {
-        const { inviteePermissionsButtonProps, inviteePermissionLevel, disabled, itemType } = this.props;
-
+        const {
+            inviteePermissions,
+            inviteePermissionsButtonProps,
+            inviteePermissionLevel,
+            disabled,
+            itemType,
+        } = this.props;
+        const defaultPermissionLevel = getDefaultPermissionLevel(inviteePermissions);
```

Comment:
Is there a reason why we can't pass `inviteePermissions` downstream from USM since we are already doing the same logic there. We can refactor this as a USM method `getInviteePermission` or something.